proposals/release-approval-process.md

```diff
@@ -10,7 +10,7 @@ This approval process hopes to encourage early consistent consensus building dur
 
 **Making a release:** OCI specification projects MUST announce intentions to release with two project maintainer sponsors (listed in the repo MAINTAINERS file) on the dev@opencontainers.org mailing list. After the announcement a two-thirds super majority of project maintainers MUST reply to the list with an LGTM within one week for the release to be approved. The maintainers MUST wait a full week for maintainers to reply but if all maintainers reply with an LGTM then the release MAY release earlier except in the case of a major releases.
 
-**Rejecting a release:** A project maintainer MAY choose to reply with REJECT and MUST include a list of concerns filed as GitHub issues. The project maintainers SHOULD try to resolve the concerns and wait for the rejecting maintainer to change their opinion to LGTM. However, a release MAY continue with a single REJECT as long two-thirds of the project maintainers approved the release.
+**Rejecting a release:** A project maintainer MAY choose to reply with REJECT and MUST include a list of concerns filed as GitHub issues. The project maintainers SHOULD try to resolve the concerns and wait for the rejecting maintainer to change their opinion to LGTM. However, a release MAY continue with REJECTs as long two-thirds of the project maintainers approved the release.
```

Comment:
On Tue, Jun 14, 2016 at 12:14:01PM -0700, Brandon Philips wrote:
> @wking good point, it should be voting maintainers. We have nearly
> hit this corner case with Vincent, Brendan, and Jon all being on
> vacation at the same time a few weeks ago.

In that case, you probably also want language establishing a quorum,
to avoid rogue rump releases ;).